<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotWindow</class>
 <widget class="QWidget" name="PlotWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>454</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>PlotWindow</string>
  </property>
  <widget class="QCustomPlot" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>601</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxXMax">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>40</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxXMin">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>10</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Graph expression:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelExpression">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>370</y>
     <width>531</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelXMin">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>58</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>X min</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelXMax">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>40</y>
     <width>58</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>X max</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelYMax">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>100</y>
     <width>58</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Y max</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelYMin">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>70</y>
     <width>58</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Y min</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxYMin">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>70</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxYMax">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>100</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAccept">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>190</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNumPoints">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>130</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of points</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxNumPoints">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>142</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
